features/page-objects/RoomSettingPage.js

```diff
@@ -36,12 +37,12 @@ class RoomSettingPage extends Page {
 
   async setAccessCode(accessCode) {
     const accessCodeInput = await this.findByCss("[id='room_access_code']");
-    accessCodeInput.sendKeys(accessCode);
+    accessCodeInput.sendKeys(accessCode.value);
```

Comment:
Maybe? Does `sendKeys` return a `Promise`? If so, then yes so that `await`s/`then`s chained onto `setAccessCode` don't accidentally execute before the `sendKeys` operation has completed.